packages/shared/util.js

```diff
@@ -8,16 +8,14 @@ export function warn (msg: string) {
   console.error(`[vue-test-utils]: ${msg}`)
 }
 
-const camelizeRE = /-(\w)/g
-export const camelize = (str: string) => str.replace(camelizeRE, (_, c) => c ? c.toUpperCase() : '')
+export function camelize (str: string): string {
+  return str.replace(/-(\w)/g, (_, c) => c ? c.toUpperCase() : '')
+}
 
-/**
- * Capitalize a string.
- */
-export const capitalize = (str: string) => str.charAt(0).toUpperCase() + str.slice(1)
+export function capitalize (str: string): string {
+  return str.charAt(0).toUpperCase() + str.slice(1)
```

Comment:
I think it's easier to read if the regex are declared outside (before refactor style).

I like the new `camelize` - but it feels like it 's more consistent with the other two helpers to do it the same way.

Is there some benefits to the refactor, or just for style?